app/assets/stylesheets/application.scss

```diff
@@ -0,0 +1,91 @@
+@import "bootstrap-sprockets";
+@import "bootstrap";
+
+body {
+    background-color: whitesmoke;
+    background-image: linear-gradient(rgb(236, 225, 248), whitesmoke);
+    font-family: Verdana, Geneva, Tahoma, sans-serif;
+  }
+
+  .nav-pills > li > a{
+    color:  rgb(78, 55, 100);
+    background-color: transparent;
+      
+  }
+
+  .nav-pills > li.active > a{
+    color: rgb(236, 225, 248);
+    background-color: rgb(78, 55, 100);
+  }
+  .nav-pills > li.active > a:hover{
+    color: rgb(236, 225, 248);
+    background-color: rgb(78, 55, 100);
+  }
+
+  .nav-pills > li > a:hover{
+    background-color: rgb(78, 55, 100);
+    color: rgb(236, 225, 248);
+  }
+
+  .navbar-brand {
+    a {
+      color: whitesmoke;
+    }
+
+  }
+
+  .flashcard {
+    width: 340px;
+    padding: 15px;
+    border: 5px solid rgb(78, 55, 100);
+    margin: 0;
+    background-color: rgb(160,148,172);
+  }
+
+  .btn {
+    
+    background-color: rgb(78, 55, 100);
+    color: whitesmoke;
+    border: solid;
+    border-width: thin;
+    
+  }
+  .btn:hover{
+    color: rgb(78, 55, 100);
+    background-color: rgb(236, 225, 248);
+  }
+
+  .navbar-default {
+    .h{
+        color: whitesmoke;
+    }
+    background-color: rgb(78, 55, 100);
+    height: 54px;
+    .navbar-nav li a {
+      color: whitesmoke;
+    }
+  }
+  
+  .navbar-container {
+    width: 800x;
+    margin: 0 auto;
+  }
+
+  html {
+    height: 100%;
+  }
+  
+  body {
+    height: 100%;
+    margin: 0;
+    background-repeat: no-repeat;
+    background-attachment: fixed;
+  }
+
+.row {
+  align-content: center;
+}
+
+.col-lg-10 {
+  align-content: center;
+}
```

Comment:
I think this is pretty self-explanatory, but I can explain this file if anyone wants me to! This is definitely not the finished stylesheet. I need to clean up the row and col-lg-10 parts though- that was part of trying to fix some rows.